zerver/views/development/registration.py

```diff
@@ -94,6 +94,7 @@ def register_demo_development_realm(request: HttpRequest) -> HttpResponse:
     realm_name = generate_demo_realm_name()
     realm_type = Realm.ORG_TYPES["business"]["id"]
     realm_subdomain = realm_name
+    email_address_visibility = UserProfile.EMAIL_ADDRESS_VISIBILITY_NOBODY
```

Comment:
We probably will ultimately want to do this in `zerver/actions` somewhere, since I don't think the intent is to use this function for demo org creation in production.

Reply:
This is a field that is configured by users (or the org default) when a person registers for a Zulip organization now.

So, I was thinking that, when we update the landing page (`/new`) to be able to create a demo organization, we would have this set in the form/button that the user clicks.

Reply:
OK, we can probably do it that way.